cms_form_compassion/controllers/payment_controller.py

```diff
@@ -13,24 +13,97 @@
 from odoo.http import request, route, Response
 from odoo.addons.website_portal.controllers.main import website_account
 from odoo.addons.cms_form.controllers.main import FormControllerMixin
+from odoo.exceptions import MissingError
+from werkzeug.exceptions import NotFound
 
 
 class PaymentFormController(website_account, FormControllerMixin):
 
+    @route(['/compassion/payment/invoice/<int:invoice_id>'], type='http',
+           website=True, methods=['GET', 'POST'],
+           auth='public', noindex=['header', 'meta', 'robots'], csrf=False)
```

Comment:
I'm not sure that putting csrf to False is a good idea, it could be a security breach. Could you remove it and test if it works without it? Otherwise we have to figure out why it fails and how we can fix that without removing the security check. Especially for payment controllers, I think it's better to be cautious.